pkg/registry/storage/csistoragecapacity/strategy.go

```diff
@@ -48,8 +48,8 @@ func (csiStorageCapacityStrategy) PrepareForCreate(ctx context.Context, obj runt
 func (csiStorageCapacityStrategy) Validate(ctx context.Context, obj runtime.Object) field.ErrorList {
 	csiStorageCapacity := obj.(*storage.CSIStorageCapacity)
 
```

Comment:
follow the same pattern of a validation options struct with a specific field controlling the validation behavior you want to drive (comment applies to all strategy.go changes)